### ISSUE TITLE
Color Codes not working in Bungee Global

### ISSUE THREAD

**Issue by Drag0nz69** [kangarko/ChatControl-Pro#13]
Version: 6.8.3

BungeeCord enabled, and using latest BungeeControlBridge

Messages sent to global (/chc global) with color codes are displaying correctly on the home server where the message originates, however, it does not format the text on the receiving servers. The color codes (ie &0-9/a-f) are broadcasted in the message without a change to text formatting.

**Comment by kangarko** [comment]
I cannot replicate.

Tested by executing /chc g and they were passed successfully, no matter if players on other server was op or not.

**Comment by Drag0nz69** [comment]
So I am running the most current version of Bungee and Spigot (1.9). ChatControl 6.8.3 and VentureChat 2.8.2 (0.6 BungeeControlBridge)

From the console or from client I execute as OP:  /chc global &cThis &bis &3a &eTest

On the server it is executed from I see: This is a Test   (In colors selected)
On the other servers in the bungee network I see: &cThis &bis &3a &eTest

Normal chatter thru VentureChat channels across Bungee are working correctly with formatting. Its just the CHC Global that is not converting to format.

Any other ideas to try, or is there more information I can provide to troubleshoot?

**Comment by Drag0nz69** [comment]
FYI - 

I setup a test server on the BungeeNetwork and removed VentureChat - so its just running ChatControPro - same results. I just wanted to eliminate VentureChat as the culprit.

**Comment by xion87** [comment]
same thing here

**Comment by kangarko** [comment]
VentureChat might cause incompatibilities especially if it servers same functionality. 

But as you stated, the issue persisted after its removal, so it can safely be excluded.

But unfortunately I am still not able to reproduce the behaviour. Tried with both operators and non-operators, everything is working on my side: http://i.imgur.com/qMwQAOZ.png

**Comment by Drag0nz69** [comment]
I updated both the Bridge and primary plugin provided yesterday, then ran the test again. I am still getting the same results, however, I did note two things I didn't notice before.
(1) I am getting a Java error on the Receiving servers
(2) The Console is showing the incoming message with formatting correct, but the MC Client is not showing the formatting but instead displays the color codes.

http://imgur.com/a/MRuol
